app/src/forms/form/service.js

```diff
@@ -307,7 +325,14 @@ const service = {
     }
   },
 
-  readSubmission: async (id) => {
+  listSubmissionFields: (formVersionId, fields) => {
```

Comment:
don't you need to flatten the array of returned objects that you got for each submission into one list of unique fields?

Reply:
ok. maybe you already did that